banyand/liaison/grpc/discovery.go

```diff
@@ -223,22 +223,25 @@ var _ schema.EventHandler = (*entityRepo)(nil)
 
 type entityRepo struct {
 	log         *logger.Logger
-	entitiesMap map[identity]partition.EntityLocator
+	entitiesMap map[identity]*partition.EntityLocator
```

Comment:
```suggestion
	entitiesMap map[identity]partition.EntityLocator
```

The receiver of methods defined in EntityLocator is the value instead of a pointer.

Reply:
Exactly. Please return an empty struct. This struct doesn't contain any large field, so instantiating it on the stack is not an overhead.